package.json

```diff
@@ -95,6 +94,7 @@
     "typescript": "^5.0.2"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "4.9.6"
+    "@openzeppelin/contracts": "4.9.6",
+    "ethers": "^6.12.0"
```

Comment:
I added a patch bump of `@nomicfoundation/hardhat-ethers` and a minor bump for `@openzeppelin/hardhat-upgrades`, but the main change here is adding `ethers` as a dependency instead of a dev dependency.